packages/lib/markdownToSafeHTML.ts

```diff
@@ -17,7 +17,8 @@ export function markdownToSafeHTML(markdown: string | null) {
     .replace(
       /<ol>/g,
       "<ol style='list-style-type: decimal; list-style-position: inside; margin-left: 12px; margin-bottom: 4px'>"
-    );
+    )
+    .replace(/<a\s+href=/g, "<a target='_blank' class='text-blue-500 hover:text-blue-600' href=");
```

Comment:
I am not very sure if tailwind would be able to detect these classes and add them accordingly but it worked during testing.